scripts/buildPublishedAPI.js

```diff
@@ -86,14 +89,25 @@ async function build() {
     }
   };
 
+  console.log('add packages to download from npm');
   // Add dependencies on each published package to the package.json, and
   // copy the docs from the current package into the temp dir.
   let packagesDir = path.join(__dirname, '..', 'packages');
   let packages = glob.sync('*/**/package.json', {cwd: packagesDir});
   for (let p of packages) {
     let json = JSON.parse(fs.readFileSync(path.join(packagesDir, p), 'utf8'));
     if (!json.private && json.name !== '@adobe/react-spectrum') {
-      pkg.dependencies[json.name] = 'latest';
+      try {
```

Comment:
if anyone has a way to query npm for many packages at once, this is incredibly slow

Reply:
this is now ameliorated because to do branch to branch comparisons we just checkout a commit directly which is pretty fast
building the published API is something we won't do very frequently, and certainly not on a branch commit build, so it's ok if this is slow